pkg/core/fs.go

```diff
@@ -82,8 +83,16 @@ func NewReadOnlyFS(bundle *Bundle, l *zap.Logger) (*ReadOnlyFS, error) {
 }
 
 // NewMutableFS creates a new instance of the datamon filesystem.
-func NewMutableFS(bundle *Bundle, pathToStaging string) (*MutableFS, error) {
+func NewMutableFS(bundle *Bundle) (*MutableFS, error) {
 	logger, _ := zap.NewProduction()
+	consumableStoreString := bundle.ConsumableStore.String()
```

Comment:
this string typing and parsing is, of course, a mess.

what's going on here is an attempt to impedance match two extant implementations:

* cli parsing

and

* mutable mount internal data structures

without changing the assumptions made by either such that the overall advertised functionality (omitting `--destination` on the cli) is in place

Reply:
:+1: for removing redundancy here

Reply:
> if we want to keep this customizable

a reason to keep this customizable is to allow fixed-size ramdisk k8s volumes to be specified in the yaml, leaning on the kubernetes system to provide storage, rather than the virtualized os within the k8s container.

> I am thinking that a `--staging` flag would be clearer for the datamon bundle mount new command

i agree that the meaning of `--destination` within `bundle mount new` is different from other commands, `bundle download`, say.  namely, because everywhere else (and i should probably double-check this) `--destination` is used, it corresponds to a directory that's modified via a discrete, one-time download operation.